samples/CMakeLists.txt

```diff
@@ -4,9 +4,17 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+
```

Comment:
Remove this

Reply:
Will fix all these small nitpicks just give me a sec :)